tests/Aspire.Hosting.Tests/Utils/TestDistributedApplicationBuilder.cs

```diff
@@ -52,6 +52,14 @@ private static IDistributedApplicationTestingBuilder CreateCore(string[] args, A
     {
         var builder = DistributedApplicationTestingBuilder.Create(args, (applicationOptions, hostBuilderOptions) => configureOptions?.Invoke(applicationOptions));
 
+        // Set a default log file name suffix based on test name for better correlation of logs to tests
+        var testName = testOutputHelper?.GetType()?.GetField("test")?.GetValue(testOutputHelper)?.ToString() ?? 
```

Comment:
This is where the use of DI container in .NET works against us--there is no easy way to determine what code will cause this function to be called. My suggestion would be to NOT default to a name based on calling assembly, as I doubt it will help us identify what test created the log. In other words, if we cannot determine the test name based on `testOutputHelper`, I would not set the suffix.

I would also not set the suffix if it is **already set**.